import-export-cli/integration/operationPolicy_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
+)
+
+const (
+	ApplicationPolicy  = "application"
+	CustomPolicy       = "custom"
+	AdvancedPolicy     = "advanced"
+	SubscriptionPolicy = "subscription"
+)
+
+// Export an API API Policy from one environment and import to another environment
+func TestExportImportAPIPolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			pathToSpecFile := testutils.DevSampleCaseOperationPolicyDefinitionPath
+			pathToSynapseFile := testutils.DevSampleCaseOperationPolicyPath
+
+			newPolicy := testutils.AddNewAPIPolicy(t, dev, user.ApiCreator.Username, user.ApiCreator.Password, pathToSpecFile, pathToSynapseFile)
```

Comment:
Are we sure that API Creator users can add new common API policies? @1akshitha are we allowing this?

Reply:
Is this related to the API Publisher?

Reply:
Fixed by [59b2698](https://github.com/wso2/product-apim-tooling/pull/925/commits/59b2698c4e9aceab8b212ba2afd761955611f7e4)